src/components/DiffView/styles.module.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../scss/variables';
+
+.diff {
+  border-bottom-left-radius: $border-radius;
+  border-bottom-right-radius: $border-radius;
+  border-collapse: separate !important;
+  border-color: $light-gray;
+  border-spacing: 0;
+  border-style: solid;
+  border-width: 0 1px 1px 1px;
+  margin-bottom: 20px;
+}
+
+.header {
+  background-color: lighten($light-gray, 2);
+  border-top-left-radius: $border-radius;
+  border-top-right-radius: $border-radius;
+  border: 1px solid $light-gray;
+  padding: $default-padding;
+}
+
+.stats {
+  font-size: 0.8em;
+  font-weight: 600;
+}
+
+.statsAdditions {
+  color: green;
```

Comment:
What about pulling this and `red` into variable so we know to change them later?

Reply:
ah, that's a good idea.